### PR TITLE
Add endpoints for ticket validation and attendant creation

### PR DESCRIPTION
**Pull request by HollowKaeden**
Implement issues #15 and #16

### CODE REVIEW

File: .gitignore
Comment (HollowKaeden) [inline .gitignore:1]: Created a virtual environment and added it to .gitignore to prevent unnecessary files from being tracked

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
-*.py[ocd]
-**/.env
-*.sqlite3
+*.py[ocd]
+**/.env
+*.sqlite3
+venv
```
Comment (oddstr13) [inline .gitignore]:
```suggestion
venv

```

File: backendapi/urls.py [hunk 2]
```diff
@@ -6,5 +6,9 @@
 
 urlpatterns = [
     path("nfctag/<hex:tag>", views.scanned),
-    path("test", views.api_get, name="api_get"),
+    path("test/", views.api_get, name="api_get"),
+    path("tickets/verify/<slug:ticket>",
+         views.verify_ticket,
+         name="verify_ticket"),
+    path("attendants/", views.create_attendant, name="create_attendant")
```
Comment (oddstr13) [inline backendapi/urls.py]:
```suggestion
    path("test", views.api_get, name="api_get"),
    path("tickets/verify/<slug:ticket>",
         views.verify_ticket,
         name="verify_ticket"),
    path("attendants", views.create_attendant, name="create_attendant")
```

no trialing slash on API paths

File: backendapi/views.py [hunk 3]
```diff
@@ -27,3 +29,42 @@ def api_get(request):
...
+    return Response(
+        {
+            "message": "Attendant created successfully.",
+            "data": {
+                "ticket_id": attendant.ticket_id,
+                "nfc_id": b16encode(attendant.nfc_id),
+                "discord": "",
+                "is_crew": attendant.is_crew,
+                "is_valid": attendant.is_valid,
+            }
```
Comment (oddstr13) [inline backendapi/views.py]:
```suggestion
            "id": attendant.id,
```

`id` is the only field that would be unknown to the client, and `discord`, `is_crew` and `is_valid` will never have anything but the default values.

We can add a return of the whole object if we find that we need it later, but then we should probably use a generator

**Review by oddstr13 — CHANGES_REQUESTED** [review]

**Review by oddstr13 — APPROVED** [review]

**Pull request merged**